externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala

```diff
@@ -149,6 +149,20 @@ class HiveOperationSuite extends HiveJDBCTestHelper {
     }
   }
 
+  test("get table types") {
+    withJdbcStatement() { statement =>
+      val resultSet = statement.getConnection.getMetaData.getTableTypes
+      val expected = Set("TABLE", "VIEW", "INDEX_TABLE", "MATERIALIZED_VIEW")
```

Comment:
hive has 2 output type
- hive: "TABLE", "VIEW", "INDEX_TABLE", "MATERIALIZED_VIEW"
- classic: "TABLE", "VIEW"

Are we ok with both of them?